v2/models.py

```diff
@@ -77,13 +77,18 @@ def __str__(self):
 class CustomerMarket(models.Model):
     customer = models.ForeignKey(Customer, on_delete=models.CASCADE, null=False)
     market = models.ForeignKey(Market, on_delete=models.CASCADE, null=False)
+    date_created = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
         return f"{self.customer.first_name} {self.customer.last_name} at {self.market.market_name}"
     
 class VendorMarket(models.Model):
     vendor = models.ForeignKey(Vendor, on_delete=models.CASCADE, null=False)
```

Comment:
Question for you - when I see models.ForeignKey is there somewhere that displays these relationships visually?  Early on, there was a data base schema, does that stay connected and update as you update the relationships?